src/converse-sendbutton.js

```diff
@@ -0,0 +1,94 @@
+// Converse.js (A browser based XMPP chat client)
+// http://conversejs.org
+//
+// Copyright (c) 2012-2017, Jan-Carel Brand <jc@opkode.com>
+// Licensed under the Mozilla Public License (MPLv2)
+//
+/*global Backbone */
+
+/* This plugin is to add an option of to have send button in chat box */
+
+(function(root, factory) {
+    define([
+        "converse-core",
+        "converse-chatview",
+        "converse-muc",
+        "converse-headline"
+    ], factory);
+}(this, function(converse) {
+    "use strict";
+    var $ = converse.env.jQuery;
```

Comment:
Please remove the jQuery dependency, we want to remove jQuery from future versions of converse.js